website/client-old/js/services/userServices.js

```diff
@@ -189,7 +189,7 @@ angular.module('habitrpg')
           }
         })
       }
-
+      
```

Comment:
There don't seem to be any functional changes committed to `userServices`, just this bit of whitespace.